core/indexer/processTransactions.go

```diff
@@ -92,7 +92,7 @@ func (tdp *txDatabaseProcessor) prepareTransactionsForDatabase(
 		if nrScResult < minimumNumberOfSmartContractResults {
 			if len(transactions[hash].SmartContractResults) > 0 {
 				scResultData := transactions[hash].SmartContractResults[0].Data
-				if strings.Contains(scResultData, "@ok") {
+				if bytes.Contains(scResultData, []byte("@ok")) {
```

Comment:
is this check ok? Could the data contain "@ok" without this being the actual status?

Reply:
yes, it should work fine. This check is needed for `esdt` calls which return only one smart contract result. And the check is ok, it will work just as the string comparison before.

Reply:
ok